R/missingness.R

```diff
@@ -134,3 +135,41 @@ countMissingData <- function(x, userNAs = NULL) {
   .Deprecated("missingness", "healthcareai")
   missingness(x)
 }
+
+
+#' Replaces missing data with NA
+#'
+#' This function replaces any given value in a dataframe with NA.
+#'
+#' @param d A dataframe or matrix
+#' @param to_replace The value that will be replaced.
+#' @return A dataframe with the missing data replaced.
+#'
+#' @export
+#' @example
+#' # Replace "NULL" in dat
+#' dat <- data.frame(a = c(1, 2, "NA", NA, "none", "??", "?", 5),
+#'                   b = c("blank", 0, "na", "None", "none", 3, 10, 4),
+#'                   c = c(10, 5, 8, 1, NA, "NULL", NaN, "Nas"),
+#'                   d = c(1, 6, 7, 8, 9, 5, 10, 1078950492))
+#'
+#' replace.missingness(dat, "NULL")
+#'
+replace.missingness <- function(d, to_replace) {
```

Comment:
please rename to `replace_missingness`.

Reply:
Yes, thank you. I am not understanding when a function classifies in the [`S3 class system`](http://adv-r.had.co.nz/S3.html). [Tidyverse](http://style.tidyverse.org/syntax.html) mentions that these are the only objects that receive a period in their naming. Any insights?

Reply:
@glenrs `x.y` is `method.class`, e.g. `plot.model_list`. You should definitely read up on this in Hadley's advanced R book: http://adv-r.had.co.nz/